### ISSUE TITLE
opam-dune-lint cannot parse dune files using `"\>`

### ISSUE THREAD

**Issue by shym** [ocurrent/opam-dune-lint#59]
Given the following simple project:

```sh
$ cat > dune-project << EOF
> (lang dune 3.8)
> (generate_opam_files)
> (name dummy)
> 
> (package
>  (name dummy)
>  (description
>   "\> Dummy
>   ))
> EOF

$ dune build dummy.opam
```

`opam-dune-lint` fails with:

```
opam-dune-lint: internal error, uncaught exception:
                Failure("Sexplib.Sexp.input_rev_sexps: reached EOF while in state Parsing_atom")
```

because the special string markers `"\>` and `"\|` are not parsed by sexplib as they are in dune (I find those markers quite useful for the long package descriptions).
Piping the dune files through `dune format-dune-file` might be a simple way, at least with dune 3.11.0 I tried, to recover a canonical string.

**Comment by edwintorok** [comment]
It also gets confused about parenthesis inside `"\>`:

  ```
    opam-dune-lint: internal error, uncaught exception:
                    (Sexplib.Sexp.Parse_error
                      ((err_msg "unexpected character: ')'") (text_line 84) (text_char 29)
                        (global_offset 2570) (buf_pos 2570)))
 ```

Which can be worked around like this, but this obviously changes the formatting in the final opam file:
```
  (name ortac-wrapper)
  (synopsis "Wrapper plugin for Ortac")
  (description
-  "\> The Wrapper plugin for the Ortac command-line tool (provided by the
-  "\> ortac-core package) can generate a wrapper module, ie a module that
+  "\> The Wrapper plugin for the Ortac command-line tool
+  "\> (provided by the ortac-core package)
+  "\> can generate a wrapper module, ie a module that
```

**Comment by moyodiallo** [comment]
opam-dune-lint uses `sexplib` https://github.com/janestreet/sexplib, to parse dune files. The [dune](https://github.com/ocaml/dune/) parser was exposed as public lib before being private, this is why the switch on `sexplib`.

opam-dune-lint doesn't need this section, it could be solved by removing description sections before parsing the `dune-project` file.

**Comment by moyodiallo** [comment]
Until a new release, you could use this fix #60.
